catalogs/sources/gtfs/schedule/fr-nouvelle-aquitaine-stga-reseau-mobius-gtfs-837.json

```
@@ -2,6 +2,11 @@
     "mdb_source_id": 837,
     "data_type": "gtfs",
     "provider": "STGA - réseau Möbius",
```

Should we deprecate this feed?
```suggestion
    "provider": "STGA - réseau Möbius",
      "status": "deprecated",
```